app/assets/stylesheets/new-capture.css

```diff
@@ -0,0 +1,570 @@
+html {
```

Comment:
Is this directly from the https://github.com/publiclab/spectral-workbench.js/ library? Could we pull its CSS in via yarn instead? And, i wonder, should the responsiveness changes happen over in that library as a result?

Reply:
Hmm, great idea ✨, should I try this in the `spectral-workbench.js` repo instead. Could this also mean that we need to update the sw-js lib the on `npm` and `yarn` dependencies...?